app/assets/stylesheets/custom.scss

```diff
@@ -364,4 +364,37 @@ h2, p {
 
 #new-cal-modal .no-padding {
   padding: 0 !important;
+}
+
+.year-row {
+  height: 25%;
+  width: 100%;
+  margin: 0;
+}
+
+.month-in-year {
+  height: 100%;
+  width: 33.3333%;
+  margin: 0;
+}
+
+.container-fluid {
+  width: 100%;
+}
+
+.year-layout {
+  width: 100%;
+  margin: 0;
+  padding: 0.7%;
+}
+
+.no-padding {
```

Comment:
Firstly, not a correct name for a class because it has padding. Secondly, I have this class(the same name) in my PR which has NO padding (`padding: 0`).

Reply:
that was the idea at first, forgot to change name